hq/app/schedule/IamMessages.scala

```diff
@@ -1,17 +1,17 @@
 package schedule
 
-import model.{AwsAccount, UserNoMfa, UserWithOutdatedKeys}
+import model.{AwsAccount, VulnerableUser}
 import org.joda.time.DateTime
 import org.joda.time.format.DateTimeFormat
 
 object IamMessages {
   def subject(account: AwsAccount) = s"Action required: Insecure credentials in the ${account.name} AWS Account scheduled for deactivaton"
   val sourceSystem = "Security HQ Credentials Notifier"
-  def outdatedKeysMessage(account: AwsAccount) =
-    s"Please rotate the following IAM access keys in AWS Account ${account.name}/${account.accountNumber} or delete them if they are disabled and unused (if you're already planning to do this, please ignore this message). If this is not rectified before the disable deadline, Security HQ will automatically disable this user:"
-  def missingMfaMessage(account: AwsAccount) =
-    s"Please add multi-factor authentication to the following AWS IAM users in Account ${account.name}/${account.accountNumber}. If this is not rectified before the disable deadline, Security HQ will automatically disable this user:"
+  def message(account: AwsAccount) =
+    s"Please check the following permanent credentials in AWS Account ${account.name}/${account.accountNumber}, which have been flagged as either needing to be rotated or requiring multi-factor authentication (if you're already planning on doing this, please ignore this message). If this is not rectified before the disable deadline, Security HQ will automatically disable this user:"
```

Comment:
not relevant to this change but I don't think "disable deadline" sounds quite right. Maybe just 'deadline'?